src/components/application_manager/src/message_helper.cc

```diff
@@ -2485,38 +2533,45 @@ bool MessageHelper::PrintSmartObject(const smart_objects::SmartObject& object) {
            key++) {
         ++tab;
 
-        printf("\n%s%s: ", tab_buffer.c_str(), (*key).c_str());
+        LOGGER_INFO(logger_,
+                    "\n" << tab_buffer.c_str() << (*key).c_str() << ": ");
         if (!PrintSmartObject(object[(*key).c_str()])) {
-          printf("\n");
+          LOGGER_INFO(logger_, "\n");
           return false;
         }
       }
       break;
     }
     case NsSmartDeviceLink::NsSmartObjects::SmartType_Boolean:
-      object.asBool() ? printf("true\n") : printf("false\n");
+      if (object.asBool()) {
```

Comment:
you can substitute this logging with single line

`LOGGER_INFO(logger_, "So value is: " << std::boolalpha << object.asBoolean())`